src/map/lua/lua_baseentity.cpp

```diff
@@ -11623,6 +11623,29 @@ inline int32 CLuaBaseEntity::spawnPet(lua_State *L)
     return 0;
 }
 
+/************************************************************************
+*  Function: spawnTrust()
+*  Purpose : Spawns a Trust if a few correct conditions are met
+*  Example : caster:spawnTrust(TRUST_SHANTOTTO)
+*  Notes   :
+************************************************************************/
+
+inline int32 CLuaBaseEntity::spawnTrust(lua_State *L)
+{
+    DSP_DEBUG_BREAK_IF(m_PBaseEntity == nullptr);
+    DSP_DEBUG_BREAK_IF(m_PBaseEntity->objtype != TYPE_PC); // only PCs can spawn trusts
+    if (!lua_isnil(L, 1) && lua_isstring(L, 1))
+    {
+        uint8 trustId = lua_tointeger(L, 1);
```

Comment:
need to explicitly cast to `uint8` or it wont compile with warnings as errors

Reply:
Done.